### ISSUE TITLE
getWSDeviceStatus returning  "Forbidden"

### ISSUE THREAD

**Issue by ttz642** [skydiver/ewelink-api#105]
Just started happening when I try to read status of devices:

V3.1

```
2|www          | 2020-09-12T10:22:03.370Z | deviceid: 10009de3ed returned error: Error: Forbidden
2|www          |     at eWeLink.getWSDeviceStatus (/home/sonoff/node_modules/ewelink-api/src/mixins/deviceControl.js:164:13) 
2|www          | 	>>> undefined
```
I'm able to getDevices() ok !

**Comment by ianhampton** [comment]
I'm seeing the same, it started yesterday.

**Comment by ttz642** [comment]
> I'm seeing the same, it started yesterday.

That's good to know it's not just me.

And up till yesterday it was OK for me also.

**Comment by skydiver** [comment]
guys @ttz642 and @ianhampton 

sorry for the looong delay
if you still experience this error, it was fixed on a new push on the 3.1 release branch
